app/controllers/application_controller.rb

```diff
@@ -1,5 +1,11 @@
 class ApplicationController < ActionController::Base
+  include Omniauth::Dsds::ControllerMethods
+
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_action :authenticate_user!
+
+  helper_method :current_user
```

Comment:
I agree that moving the `helper_method` defn into the gem would make sense. Wouldn't moving the `before_action` in too obscure a huge part of every controller action? I'm not sure I like the idea of having this invisible `before_action` being hidden away?